### DISCUSSION TITLE
ToDoes projects and contexts are sorted at the end of the line

### DISCUSSION THREAD

**Discussion by glurmito** [ransome1/sleek#456] [Q&A] [answered]
Hello.

Firstly I must say thanks for this excellent TodoTXT editor. I'm really a fan of this format and by far this is most complete and functional application.

Because I've started using it recently maybe I'm just missing the configuration that controls this behaviour so apologies in advance.

The thing is that when I add a ToDo the project and the context tags are displayed always at the end of the line breaking any intent to use "natural language" in the tagging.

For instance: "During +spring_cleaning clean the @\kitchen and the @\garaje" is displayed as: "During clean the and the +spring_cleaning @\kitchen @\garaje".

(The contexts have been escaped to prevent mentions)

Is if possible to disable this feature and show the tags inline?

Thank you for your time.

**Comment by CPUGPU** [comment, marked as answer]
Hi glurmito.  There is an open issue https://github.com/ransome1/sleek/issues/68 covering your request exactly.  Unfortunately it relies on an upstream library that has not implemented that feature yet.

**Reply by kitchen** [reply]
Careful with the @ mentions please :)
